app/assets/stylesheets/master.css.scss

```diff
@@ -0,0 +1,3 @@
+.clear{
+	clear: both;

```

Comment:
this works also worth understanding. http://nicolasgallagher.com/micro-clearfix-hack/